src/structures/User.js

```diff
@@ -179,17 +179,19 @@ class User extends Base {
    * @readonly
    */
   get hexAccentColor() {
-    if (!this.accentColor) return null;
+    if (!this.accentColor) return this.accentColor;
     return `#${this.accentColor.toString(16).padStart(6, '0')}`;
```

Comment:
```suggestion
    return this.accentColor && `#${this.accentColor.toString(16).padStart(6, '0')}`;
```
i believe this would work too (removing the above line also)

Reply:
actually on second thoughts

if accentColor is 0 then 0 would be returned (in both instances) so perhaps a more robust check is required

Reply:
I can probably just add a check if accentColor isn’t a number on the first one